std_msgs/Header header
string time
fivecomm_5g_router/techInfo infotech
fivecomm_5g_router/threegInfo info3g
fivecomm_5g_router/fourgInfo info4g
fivecomm_5g_router/fivegInfo info5g


